.circleci/config.yml

```
@@ -201,41 +279,47 @@ jobs:
           command: mkdir -p ./test/results
       - run:
           name: Execute integration tests
-          command: RESULTS_DIR="./test/results" npm -s run test:integration
+          command:  RESULTS_DIR="./test/results" npm -s run test:integration
```

There is an issue with `default-machine` that it does not auto-select the NVM version. Thus the work-around is to add a `nvm use default` for now...

```suggestion
          command:  |
            # Set Node version to default (Note: this is needed on Ubuntu)
            nvm use default
            
            # Execute Integration Tests
            RESULTS_DIR="./test/results" npm run test:integration
```

I also removed the `-s` to be consistent with other repos.